go.mod

```diff
@@ -11,16 +11,16 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.29.9
 	github.com/aws/aws-sdk-go-v2/service/sts v1.33.17
 	github.com/coreos/go-oidc/v3 v3.13.0
-	github.com/envoyproxy/gateway v0.5.0-rc.1.0.20250305132535-64b753bfa5fd
+	github.com/envoyproxy/gateway v0.5.0-rc.1.0.20250318185001-4143f5c8eb2d
 	github.com/envoyproxy/go-control-plane/envoy v1.32.5-0.20250303232259-0c5cb90447bb
 	github.com/go-logr/logr v1.4.2
-	github.com/google/cel-go v0.24.1
+	github.com/google/cel-go v0.22.1
```

Comment:
this was needed due to https://github.com/kubernetes/apiserver/issues/116 and need to pin it to 0.22.x until it's resolved